Room filter

yashka713 opened the pull request:
Added and realised room-filter

Realised filter dor room by `price', `pccupancy`, `room type` and `date`

yashka713 committed:
Initial commit for home controller

yashka713 committed:
Fixed controller name and all related resourses

yashka713 committed:
Remove unused route

yashka713 committed:
Create logik for filter in homes controller

yashka713 committed:
Correct logik for filter, without date

yashka713 committed:
Add scopes and realised filtering

Add scopes for Apartment

Add apartment_filter as model for filtering info by sended data

Validates field in apartment_filter

Vitaliy-Maznyi committed:
add view for homes index

add filter form, results blocks, ApartmentsFilterForm, bootstrap js slider

Vitaliy-Maznyi committed:
resolve conflicts

Vitaliy-Maznyi committed:
add occupancy increment

yashka713 committed:
Created filter for Apartments

Vitaliy-Maznyi committed:
update filter form view and add vendor animate.css

add spans with min/max values, add blank to inputs, add animate.css

yashka713 committed:
Merge remote-tracking branch 'origin/appartments-filter' into appartments-filter

yashka713 committed:
Unit of controller and views

Added default value for occupancy

Added helper for FilterForm form object

yashka713 committed:
Added seeds for Apartments and Users

yashka713 committed:
Added filter service for rooms

Added filters for price, date, occupancy, room type

yashka713 committed:
Remove unused scope and filter example

yashka713 committed:
Added endline

yashka713 committed:
Added Controller and Form Object for room filter

yashka713 committed:
Merge branch 'master' into appartments-filter

yashka713 committed:
Rebuild schema.rb and Gemfile.lock to resolve error with bundle

Vitaliy-Maznyi commented on app/controllers/homes_controller.rb:
```diff
@@ -0,0 +1,17 @@
+require 'ApartmentsFilterForm.rb'
```
why do you need to require this file?

Vitaliy-Maznyi commented on app/models/ApartmentsFilterForm.rb:
```diff
@@ -0,0 +1,29 @@
+  end
+
+  def price_from
+    price = :price
```
Why do you do this on line 24?

yashka713 replied:
Because `:price.to_s` is not working. And method `split` absent for symbol.

Vitaliy-Maznyi replied:
why can't you use just `price`? Try to comment this line and test behaviour

yashka713 committed:
Added hidden field for :order 

Now we can make order from start page

All the necessary information we get from the data fields

yashka713 committed:
Added count price of rented room

yashka713 committed:
Pass data from user input field into hidden field of order

yashka713 committed:
Merge branch 'master' into appartments-filter

yashka713 committed:
Removed unused string

yashka713 committed:
Merge remote-tracking branch 'origin/appartments-filter' into appartments-filter
